From 404af5e1d854522df2b02db9f8805ce19f9e8b76 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 22 Oct 2010 15:28:40 +0000
Subject: Handle CC="ccache gcc" for linker command too

The compiler path explicitly split()s the variable, we should do
so in the linker path too.
---
diff --git a/giscanner/dumper.py b/giscanner/dumper.py
index 773dec9..7462aa3 100644
--- a/giscanner/dumper.py
+++ b/giscanner/dumper.py
@@ -200,7 +200,8 @@ class DumpCompiler(object):
             args.append('--tag=CC')
             args.append('--silent')
 
-        args.extend([self._linker_cmd, '-o', output])
+        args = self._linker_cmd.split()
+        args.extend(['-o', output])
         if libtool:
             args.append('-export-dynamic')
 
--
cgit v0.8.3.1
